cibuild.cmd

```diff
@@ -42,8 +42,8 @@ if not "%BuildTimeLimit%" == "" (
 ) else (
     set RunProcessWatchdog=false
 )
-    
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat" || goto :BuildFailed
+
+call "%RoslynRoot%SetDevCommandPrompt.cmd" || goto :BuildFailed
```

Comment:
I think it's okay to make Dev15 the default if Dev15 is installed.  But we should also have a way to force Dev14 to be used.  

Also: this should be in `build/scripts` vs. the root directory.